components/models/entity.py

```diff
@@ -0,0 +1,50 @@
+import csv
+import json
+import os
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+CONFIG_FILE_PATH = os.path.join(BASE_DIR, 'config', 'globalConfig.json')
+
+class Entity:
+
+    def __init__(self, Reference=None, Name=None, Geometry=None, Point=None, Start_date=None, End_date=None, Notes=None, Legislation=None, Document_url=None):
+        self.Reference = Reference
+        self.Name = Name
+        self.Geometry = Geometry
+        self.Point = Point
+        self.Start_date = Start_date
+        self.End_date = End_date
+        self.Notes = Notes
+        self.Legislation = Legislation
+        self.Document_url = Document_url
```

Comment:
doesn't matter much for this push, but good to note that we will have to dynamically assign attributes. as not all datasets will have the same column names. but that can be done in a later PR

Reply:
then again we load the data from the csv and don't use the constructor with args so perhaps this doesn't matter?